File: app/src/at-vinhnguyen/java/asiantech/internship/summer/javacore/exercise14/ListElementsThatAppearOnly.java
package asiantech.internship.summer.javacore.exercise14;

import android.util.Log;

import static asiantech.internship.summer.javacore.InputUtils.inputIntNumber;

public class ListElementsThatAppearOnly {

    private static final String TAG = ListElementsThatAppearOnly.class.getSimpleName();

    private static int countElement(int a[], int n, int i) {
        int count = 0;
        for (int j = 0; j < n; j++) {
            if (a[j] == i) {
                count++;
            }
        }
        return (count);
    }

    public static void main(String[] args) {
        int n, i;
        Log.d(TAG, "Nhap n= ");
        n = inputIntNumber(TAG);
        int[] array = new int[n];
        for (i = 0; i < n; i++) {
            Log.d(TAG, "Nhap phan tu thu " + (i + 1) + " ");
            array[i] = inputIntNumber(TAG);
        }
        Log.d(TAG, "Cac phan tu trong day xuat hien 1 lan: ");
        for (i = 0; i < n; i++) {
            if (countElement(array, n, array[i]) == 1) {
                Log.d(TAG, String.valueOf(array[i]));
            }
        }
    }
}

File: app/src/at-vinhnguyen/java/asiantech/internship/summer/kotlincore/exercise14/ListElementsThatAppearOnly.kt
package asiantech.internship.summer.kotlincore.exercise14

import asiantech.internship.summer.kotlincore.inputIntNumber

object ListElementsThatAppearOnly {

    private fun countElement(a: IntArray, n: Int, i: Int): Int {
        var count = 0
        for (j in 0 until n) {
            if (a[j] == i) {
                count++
            }
        }
        return count
    }

    @JvmStatic
    fun main(args: Array<String>) {
        val n = inputIntNumber()
        var i = 0
        val array = IntArray(n)
        while (i < n) {
            print("Nhap phan tu thu " + (i + 1) + " ")
            array[i] = inputIntNumber()
            i++
        }
        print("Cac phan tu trong day xuat hien 1 lan: ")
        i = 0
        while (i < n) {
            if (countElement(
                            array, n, array[i]) == 1) {
                print("  " + array[i])
            }
            i++
        }
    }
}
